### DISCUSSION TITLE
我问它的资料库是什么时候的、它说是2020年的、官网是2021年的、所以它确定是GPT-3.5吗

### DISCUSSION THREAD

**Discussion by yuzyf** [acheong08/ChatGPT#708] [Q&A] [answered]
<img width="565" alt="image" src="https://user-images.githubusercontent.com/77232395/218465186-caeed917-36ca-49f2-ba6c-0708ca3d1e96.png">

**Comment by yuzyf** [comment]
<img width="431" alt="image" src="https://user-images.githubusercontent.com/77232395/218466342-daab4fc5-4595-48eb-8c17-b0a5c1f9b0b2.png">
<img width="518" alt="image" src="https://user-images.githubusercontent.com/77232395/218466474-4940f609-bbfa-4ff7-9d45-257bf6db313f.png">
官网是截止到2021年、所以它确定是gpt-3.5吗

**Comment by yuzyf** [comment]
使用的是revChatGPT.V2

**Comment by lss233** [comment]
AI 不知道自己叫什么，这种回答参考价值不大。

**Reply by yuzyf** [reply]
那知道如何判断它是不是GPT-3.5呢

**Reply by acheong08** [reply]
The conversation shows up on chat.openai.com

**Comment by VacantHusky** [comment]
我问一个小孩子他哪一年出生，他也会瞎回答一个答案。
openai官网提供的使用key访问api的方式是用的GPT-3而非GPT-3.5。这个项目中并没有让我们提供key，而是账号和密码，模拟登录，用的就是网页上的chatGPT。

**Comment by lp123sun** [comment, marked as answer]
Chatbot: 
我的知识库截止日期是2021年9月，但我会不断地学习和更新我的知识，以保持尽可能的最新性。当前的日期是2023年2月16日，所以我现在的知识应该比截止日期更加丰富。
